internal/stacks/stack_manager.go

```diff
@@ -188,6 +188,11 @@ func (s *StackManager) InitStack(options *types.InitOptions) (err error) {
 	if err := s.writeDockerComposeOverride(compose); err != nil {
 		return fmt.Errorf("failed to write docker-compose.override.yml: %s", err)
 	}
+	if options.CustomPath == "docker-compose.yml" {
```

Comment:
I think this check now enforces that you can only ever pass in a file named `docker-compose.yml` in the current directory. I would think we would just want to check to see if this is an empty string here (and then maybe validate the file exists).